simularium/binding-sim-edu
# Feature/resize

Problem
=======
Estimated review size: medium

If you resize the window it doesn't update the simulation viewport 
closes #29 

Solution
========
Added an event listener that sends the new size into the client simulator 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)



Steps to Verify:
----------------
1. switch the simulation 
2. resize the window 
3. the simulation should reset to the new window size 

Screenshots (optional):
-----------------------
<img width="1636" alt="Screenshot 2024-04-05 at 11 38 47 AM" src="https://github.com/simularium/binding-sim-edu/assets/5170636/ec8c54ac-cfe7-45fc-852f-e85f51904ac7">
